skrub/tests/test_table_vectorizer.py

```diff
@@ -778,11 +777,7 @@ def test_drop_null_column():
 
     # Raise exception if a null column is found
     with pytest.raises(
```

Comment:
This test is still failing because the TableVectorizer is not raising the correct exception and I don't know how to make it do that.

Reply:
[here](https://github.com/skrub-data/skrub/pull/1115/files#diff-915f4c6f453724cbd65c06d5c7f55a1aed027f2b5649d15ec4ec3b9edb90ad70R77) raise a ValueError instead of RejectColumn

Reply:
rejectcolumn is a way to signify to the tablevectorizer "I'm not the right transformer for this column, don't apply me here".

Reply:
Gotcha, fixed